Calmspace/DetailsTableViewCell.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  DetailsTableViewCell.swift
+//  Calmspace
+//
+//  Created by Admin on 12/10/2023.
+//
+
+import UIKit
+
+class DetailsTableViewCell: UITableViewCell {
+
+    @IBOutlet weak var detailsLabel: UILabel!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
```

Comment:
This code does nothing (in object-oriented programming, if the child class does not override a method, the parent implementation is called; here, the same thing is done: we only call `super`). The code that does nothing has to be removed.